.readthedocs.yaml

```diff
@@ -0,0 +1,21 @@
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+
+
+version: 2
+build:
+  os: "ubuntu-20.04"
+  tools:
+    python: "miniconda3-4.7"
+conda:
+  environment: conda-recipe.yaml
```

Comment:
could not build without environment variable: CONDA_OVERRIDE_CUDA=11.7 @XuehaiPan